CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+CHANGES IN VERSION 1.16.0 (FROM 1.15.1)
+===================================
+
+Appium 1.16.0 is a minor release
+
+#### General
+
+#### IOS (XCUITest):
```

Comment:
* Add support for shadow DOM elements (https://github.com/appium/appium-remote-debugger/commit/559395a16088142b27289dbac6d3a5ab36caa716)
* fix problem in webview asynchronous execute on older versions of iOS, when low timeouts would cause the return value to never be returned (https://github.com/appium/appium-remote-debugger/pull/185)